app/controllers/application_controller.rb

```diff
@@ -36,7 +36,8 @@ class ApplicationController < ActionController::Base
   helper_method :current_templates
   helper_method :admin?, :logged_in?, :highlight_all
 
-  #  protect_from_forgery :secret => '112141be0ba20082c17b05c78c63f357'
+  protect_from_forgery with: :exception
```

Comment:
Correct me if I'm wrong, But when you turn on forgery protection in ApplicationController. It is turned on in the whole app. So when a controller/integration test makes a request in test ENV. whole application is loaded and request is processed (just as it would get processed in development ENV) with same code but just in a test ENV.  

So I believe forgery protection is on in the test suites as well.